### PR TITLE
Add Honeywell Lyric Component Docs

### PR DESCRIPTION
**Pull request by timmo001**
**Description:**

Adds docs for new component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23015

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by timmo001** [comment]
Not sure why netlify didn't generate a page at:

https://deploy-preview-9201--home-assistant-docs.netlify.com/components/lyric/

Did something change, did I miss something or is this a bug I've hit?

**Comment by klaasnicolaas** [comment]
That happens when you use the `redirect_from` for the same page 😆 It works now 😉

File: source/_components/lyric.markdown [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+1. Once your app is created, copy the `Consumer Key` and `Consumer Secret`. These will be your `client_id` and `client_secret`.
+
+## {% linkable_title Configuration %}
+
+To enable this component, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+lyric:
+  client_id: !secret lyric_client_key
```
Comment (klaasnicolaas) [inline source/_components/lyric.markdown]: At the moment we have not a kind of policy, that you must use `!secret` (ofcourse it's better 😉 ). But the examples are for simple copy/paste work.

**Comment by timmo001** [comment]
Ah. Thanks!

**Review by klaasnicolaas — DISMISSED** [review]

**Comment by klaasnicolaas** [comment]
Is there some progress?

File: source/_components/lyric.markdown [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: honeywell.png
+ha_category:
+  - Hub
+  - Climate
+  - Sensor
+ha_release: 0.95
```
Comment (klaasnicolaas) [inline source/_components/lyric.markdown]:
```suggestion
ha_release: 0.96
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]

**Pull request closed without merging**